<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResizableViewDialog</class>
 <widget class="QDialog" name="ResizableViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>420</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Загрузить изображение</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CustomWidgetsQt::ResizableView" name="resizableView" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="loadButton">
       <property name="toolTip">
        <string>Загрузить</string>
       </property>
       <property name="statusTip">
        <string>Загрузить</string>
       </property>
       <property name="whatsThis">
        <string>Загрузить</string>
       </property>
       <property name="text">
        <string>Загрузить</string>
       </property>
       <property name="iconSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearButton">
       <property name="toolTip">
        <string>Удалить</string>
       </property>
       <property name="statusTip">
        <string>Удалить</string>
       </property>
       <property name="whatsThis">
        <string>Удалить</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="iconSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="resizeButton">
       <property name="toolTip">
        <string>Изменить размер изображения</string>
       </property>
       <property name="statusTip">
        <string>Изменить размер изображения</string>
       </property>
       <property name="whatsThis">
        <string>Изменить размер изображения</string>
       </property>
       <property name="text">
        <string>Размер</string>
       </property>
       <property name="iconSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="applyButton">
       <property name="toolTip">
        <string>Применить и закрыть</string>
       </property>
       <property name="statusTip">
        <string>Применить и закрыть</string>
       </property>
       <property name="whatsThis">
        <string>Применить и закрыть</string>
       </property>
       <property name="text">
        <string>Применить и закрыть</string>
       </property>
       <property name="iconSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomWidgetsQt::ResizableView</class>
   <extends>QWidget</extends>
   <header>CustomWidgets/ResizableView/resizableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
